src/ringct/rctSigs.cpp

```diff
@@ -44,6 +44,19 @@ using namespace std;
 
 #define CHECK_AND_ASSERT_MES_L1(expr, ret, message) {if(!(expr)) {MCERROR("verify", message); return ret;}}
 
+namespace
+{
+    rct::Bulletproof make_dummy_bulletproof(size_t n_outs)
+    {
+        const rct::key I = rct::identity();
+        size_t nrl = 0;
+        while ((1 << nrl) < n_outs)
```

Comment:
This triggers a signed and unsigned integer comparison when running with -Werror=sign-compare .